package.json

```diff
@@ -54,7 +54,8 @@
     "test.testing": "node scripts/test/validate-testing.js",
     "test.prod": "npm run test.dist && npm run test.end-to-end && npm run test.jest && npm run test.karma && npm run test.sys.node && npm run test.testing && npm run test.analysis",
     "test.watch": "jest --watch",
-    "test.watch-all": "jest --watchAll --coverage"
+    "test.watch-all": "jest --watchAll --coverage",
+    "lint": "eslint src/*.ts src/**/*.ts src/**/*.tsx"
```

Comment:
Nit - can we keep this list of scripts alphabetized by their name/key?

Reply:
Well, I guess I can't read - cause I thought this was already alphabetized 🙃 thanks!